acceptance-tests/src/test/java/gov/di_ipv_drivingpermit/step_definitions/WelshlangDrivingLicenceStepDefs.java

```diff
@@ -0,0 +1,567 @@
+
+    @And("I can see the sentence “Gadewch hyn yn wag os nad oes gennych unrhyw enwau canol”")
+    public void iCanSeeTheSentenceGadewchHynYnWagOsNadOesGennychUnrhywEnwauCanol() {
+        middleNameSentence();
+    }
+
+    @Given("I can see the DoB fields titled “Dyddiad geni”")
+    public void iCanSeeTheDoBFieldsTitledDyddiadGeni() {
+        DateOfBirthField();
+    }
+
+    @And("I can see example as Er enghraifft")
+    public void iCanSeeExampleAsErEnghraifft() {
+        DateOfBirthFieldhint();
+    }
+
+    @And("I can see date as “Diwrnod”")
+    public void iCanSeeDateAsDiwrnod() {
+        dateField();
+    }
+
+    @And("I can see date for DVA as “Diwrnod”")
+    public void iCanSeeDateForDVAAsDiwrnod() {
+        dateFieldDVA();
+    }
+
+    @And("I can see month as “Mis”")
+    public void iCanSeeMonthAsMis() {
+        monthField();
+    }
+
+    @And("I can see month for DVA as “Mis”")
+    public void iCanSeeMonthForDVAAsMis() {
+        monthFieldDVA();
+    }
+
+    @And("I can see year as “Blwyddyn”")
+    public void iCanSeeYearAsBlwyddyn() {
+        yearField();
+    }
+
+    @Given("I can see the Issue date field titled “Dyddiad cyhoeddi”")
+    public void iCanSeeTheIssueDateFieldTitledDyddiadCyhoeddi() {
+        issueDateField();
+    }
+
+    @Then("Dyma r dyddiad yn adran 4a o ch trwydded, er enghraifft 27 5 2019")
+    public void dymaRDyddiadYnAdranAOChTrwyddedErEnghraifft() {
+        issueDateSentence();
+    }
+
+    @Then("Dyma r dyddiad yn adran 4b o ch trwydded, er enghraifft 27 5 2019")
+    public void dymaRDyddiadYnAdranAOChTrwyddedErEnghraifftValidityDate() {
+        validityDateSentence();
+    }
+
+    @Then("I can see the Valid to date field titled “Yn ddilys tan”")
+    public void iCanSeeTheValidToDateFieldTitledYnDdilysTan() {
+        validToDateFieldTitle();
+    }
+
+    @Given("I can see the licence number field titled “Rhif trwydded”")
+    public void iSelectedDVLAOnThePreviousPage() {
+        licenceNumberWelsh();
+    }
+
+    @Given("I can see the licence number field for DVA titled “Rhif trwydded”")
+    public void iSelectedDVLAOnThePreviousPageForDVA() {
+        licenceNumberWelshDVA();
+    }
+
+    @And("I see the sentence “Dyma'r rhif hir yn adran  ar eich trwydded”")
+    public void iSeeTheSentenceDymaRRhifHirYnAdranArEichTrwydded() {
+        licenceSentence();
+    }
+
+    @Then("I can see the issue number field titled “Rhif cyhoeddi”")
+    public void iCanSeeTheIssueNumberFieldTitledDyddiadCyhoeddi() {
+        issueNumberWelsh();
+    }
+
+    @And("I  can see “Dyma r rhif  ddigid ar ôl y gofod yn adran  o'ch trwydded”")
+    public void iCanSeeDymaRRhifDdigidArÔlYGofodYnAdranOChTrwydded() {
+        issueNumberSentence();
+    }
+
+    @Then("I can see the postcode field titled “Cod post”")
+    public void iCanSeeThePostcodeFieldTitledCodPost() {
+        postcodeWelsh();
+    }
+
+    @And("I can see “Rhowch y cod post yn y cyfeiriad yn adran  o ch trwydded”")
```

Comment:
@chinmayi-gds @MathumithaVe should all these I can see methods not take an argument rather than a hard coded string